### PR TITLE
Updated `README.md` with project narrative

### PR DESCRIPTION
**Pull request by fuxingloh**
#### What kind of PR is this?:

/kind docs

#### What this PR does / why we need it:

- Updated CODEOWNERS, governance and `.github/*` with new project structure
- Updated `README.md` with 
  - project structure
  - releases environment
  - testing

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,6 +56,69 @@ up [npm workspaces](https://docs.npmjs.com/cli/v7/using-npm/workspaces).
...
+6. `/store`  - global state that is used at least more than once in screens
+7. `/translations` - various language translations
+
+### Testing
+
+There are 2 types of tests in DeFi Wallet.
+
+#### Unit Testing
+
+Unit testing are created to test each individual units/components of a software. As they are unit tests, they should be
```
Comment (thedoublejay) [inline README.md:94]:
```suggestion
Unit testing is created to test each individual units/components of a software. As they are unit tests, they should be
```

File: README.md [hunk 2]
```diff
@@ -8,6 +8,42 @@
...
 
+## Releases
+
+DeFi Wallet has 3 releases channel and unique environment for each of those
+channel. [`app/environment.ts`](/app/environment.ts) carries the environment state for those releases channel.
+
+<details>
+<summary><b>Production</b></summary>
+
+Created by Expo Application Service and configured in [`eas.json`](/eas.json) it creates a native build
```
Comment (thedoublejay) [inline README.md:19]:
```suggestion
Created by Expo Application Service and configured in [`eas.json`](/eas.json), It creates a native build
```

File: README.md [hunk 3]
```diff
@@ -8,6 +8,42 @@
...
+with [`release-publish.yml`](/.github/workflows/release-publish.yml) workflow on type "published". Builds can only be
+trigger by DeFiChain engineers, they are automatically uploaded into native app store for distribution.
```
Comment (thedoublejay) [inline README.md:21]:
```suggestion
triggered by DeFiChain engineers, they are automatically uploaded into native app store for distribution.
```

File: README.md [hunk 4]
```diff
@@ -8,6 +8,42 @@
...
+
+Production environment is no debug and **MainNet** is the only network available.
```
Comment (thedoublejay) [inline README.md:23]: This line is a bit confusing. Maybe something like, In production environment, only **MainNet** is available and debugging is not available.

**Review by thedoublejay — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (fuxingloh) [reply README.md:19]:
```suggestion
Created by Expo Application Service and configured in [`eas.json`](/eas.json), it creates a native build
```

File: README.md (hunk 4 above)
Comment (fuxingloh) [reply README.md:23]:
```suggestion
In the production environment, only MainNet is available, and debugging is not available.
```

File: README.md [hunk 5]
```diff
@@ -8,6 +8,42 @@
...
+</details>
+
+<details>
+<summary><b>Preview</b></summary>
+
+Preview builds are created by 2 workflow. First at each pull request
+via [`expo-preview.yml`](/.github/workflows/expo-preview.yml) workflow, release are prefixed `pr-preview-`. Secondly
+at  [`release-publish.yml`](/.github/workflows/release-publish.yml) workflow on type "prereleased".
+
+Preview environment has debug ability and all available network configured.
```
Comment (fuxingloh) [inline README.md:34]:
```suggestion
In the preview environment, all networks are available, and debugging is enabled.
```

File: README.md [hunk 6]
```diff
@@ -8,6 +8,42 @@
...
+
+</details>
+
+<details>
+<summary><b>Development</b></summary>
+
+Development builds are created on local machine and not triggered by any CI workflow.
+
+Development environment has debug ability and all playground network configured.
```
Comment (fuxingloh) [inline README.md:43]:
```suggestion
In the development environment, all playground networks are available, and debugging is enabled.
```

**Review by thedoublejay — APPROVED** [review]

**Pull request merged**